src/runtime_src/core/pcie/driver/linux/xocl/lib/libxdma.c

```diff
@@ -4097,6 +4097,9 @@ MODULE_AUTHOR("Xilinx, Inc.");
 MODULE_DESCRIPTION(DRV_MODULE_DESC);
 MODULE_VERSION(DRV_MODULE_VERSION);
 MODULE_LICENSE("GPL v2");
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,16,0) || defined(RHEL_9_0_GE)
```

Comment:
@Bhyaswanth Why do you need to import DMA_BUF? If you remove this, do you see any errors?

Reply:
Yes @sureshreddyavula , We are seeing the DKMS configure issue while installing XRT on 6.2 kernel.